test/hibp.test.js

```diff
@@ -162,6 +163,22 @@ describe('hibp', () => {
     });
   });
 
+  describe('breachedAccount (unexpected HTTP error)', () => {
+    it('should throw an Error width the error text from the request', () => {
```

Comment:
"width" should be "with" (typo), but let's use this now anyway:

* `'should throw an Error with the status text from the response'`